app/test/integration_test.go

```diff
@@ -225,9 +225,10 @@ func (s *IntegrationTestSuite) TestSubmitPayForBlob() {
 	assert := s.Assert()
 	val := s.network.Validators[0]
 	ns1 := appns.MustNewV0(bytes.Repeat([]byte{1}, appns.NamespaceVersionZeroIDSize))
+	share1 := uint8(0)
```

Comment:
Usages of `share1` in this file can be replaced with `appconsts.ShareVersionZero`

```suggestion
```

Reply:
done